bin/themes/Dark/style.css

```diff
@@ -3,70 +3,70 @@
 /* Use 'url(./xxxx)' for resources, it will be replaced with the right path at runtime */
 
 ShortcutEdit[error="true"] {
-    color: #EF5350;

```

Comment:
With all these whitespace changes it's not possible to see what you actually changed. Please configure your editor to not reformat the whole file.